pkg/collector/corechecks/ebpf/c/runtime/ebpf-kern.c

```diff
@@ -7,6 +7,7 @@
 #include "bpf_metadata.h"
 
 #define F_DUPFD_CLOEXEC 1030
+#define EBPF_CHECK_KPROBE_MISSES_CMD 0x70C14
```

Comment:
A later PR will provide some object to manage bpf-on-ioctl programs. The library function will take care of assigning relevant ioctl commands so that we do not have to have these magic values across the code base.